docs/foundations/icons.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Icons
+---
+
+## Intro
+
+SF DesSys icons are visual representations of ideas and actions. These icons allow our users to quickly see interface intent at a glance and take part in establishing hierarchy.
+
+Icons also assist with accessibility in scenarios of low literacy or English proficiency.
+
+## Principles
+
+Clarity
+
+Culturally aware
+
+*Add potential principles here to be fleshed out
+
+## Design Guidelines
+
+### Interface Icons vs. Picture Icons
+
+The San Francisco Design System utilizes two types of images to assist users; interface icons and Picture icons.
+
+Interface icons, or just icons, are generally used to convey an action or intent and are not illustrative or decorative.
+
+Picture icons serve some of the same purposes as the interface icons but in a more expressive way.
+
+### Deciding on a new icon
+
+Before creating a new icon, pause and look for an icon in the SF DesSys icon library that clearly conveys your message; there’s no reason to bloat the icon library for the sake of having variety.
+
+![A calendar icon being used with the word Events](/assets/need-for-new-icon-1.png)
```

Comment:
eleventy does some under-the-hood magic that means that accessing the images isn't as simple as reaching into the directory you're working in. (this is why the images aren't showing up in the deployment rn) I _think_ what we'll wanna do is shove images into a `static/` directory or find a way to put it into the `dist/` dir. maybe we can discuss next week and pair/trio on it if necessary. [here's](https://www.11ty.dev/docs/copy/) the not-so-straightforward way of getting files into the build